code/datums/attributes/_attribute.dm

```diff
@@ -73,6 +89,15 @@ GLOBAL_LIST_INIT(attribute_types, list(
 		return 1
 	return max(1, atr.get_level())
 
+//Getting modified level, which is level + buff + bonus
+/proc/get_modified_attribute_level(mob/living/carbon/human/user, attribute)
+	if(!istype(user) || !attribute)
+		return 1
+	var/datum/attribute/atr = user.attributes[attribute]
+	if(!istype(atr))
+		return 1
```

Comment:
Why not FALSE / 0? This would allow for checking if an attribute/person even exists

Reply:
The original attribute-related procs follow this same methodology, for a good reason I presume. I'm not the person who originally wrote them, I could look at doing it for all of them if it doesn't break things.

Reply:
It is to avoid divisions by 0 in certain places.